src/appBootstrapper.jsx

```diff
@@ -1,40 +1,41 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from 'appShell/App/App';
-import { hashHistory, createMemoryHistory } from 'react-router';
-import { configureStore } from './redux/configureStore';
+import { Provider } from 'mobx-react';
+import { hashHistory, Router } from 'react-router';
+import { RouterStore, syncHistoryWithStore  } from 'mobx-react-router';
+import ExtendedRoutingStore from './shared/lib/ExtendedRouterStore';
+import { computed, extendObservable } from 'mobx';
 import makeRoutes from './routes';
 import lodash from 'lodash';
 
+let RT = require('route-parser');
```

Comment:
Actually, this is a library which does route parsing and construction.  So it's possible it could be choice for your comment below.  I had some reason for moving away from it.  Not sure.